install/pre-install-payload/scripts/reqs-deploy.sh

```diff
@@ -4,6 +4,11 @@ set -o errexit
 set -o pipefail
 set -o nounset
 
+INSTALL_COCO_CONTAINERD=${INSTALL_COCO_CONTAINERD:-false}
+INSTALL_OFFICIAL_CONTAINERD=${INSTALL_OFFICIAL_CONTAINERD:-true}
+INSTALL_VFIO_GPU_CONTAINERD=${INSTALL_VFIO_GPU_CONTAINERD:-false}
+INSTALL_NYDUS_SNAPSHOTTER=${INSTALL_NYDUS_SNAPSHOTTER:=true}
```

Comment:
@fidencio I don't know this ':=' syntax but for some reason it works. Is it intentional or a typo?

Reply:
No, that's an unintentional typo. :-)